content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -1648,43 +1648,75 @@ The `header` element accepts the following children elements:
 
 .. _reference/view_architectures/list/control:
 
-`control` & `create`: add inline create buttons
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`control`: customize create and delete actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The `control` element defines a control row that accepts create buttons. Each create button is
-defined through a `create` element.
```

Comment:
I would keep an intro paragraph because the purpose of the `control` element is not obvious IMHO.